backend/src/main/java/de/jannisaziz/backend/models/SavedGame.java

```diff
@@ -0,0 +1,21 @@
+package de.jannisaziz.backend.models;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class SavedGame {
```

Comment:
SavedGame? Do notSavedGames exist?

Reply:
This is a game that the user adds/saves to their library